scriptmodules/libretrocores/lr-superflappybirds.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="lr-superflappybirds"
+rp_module_desc="Super Flappy Birds - Multiplayer Flappy Bird Clone"
+rp_module_help="Checkout the wiki on my github page."
```

Comment:
Just one final remark, can you provide the actual URL to your wiki here? Thanks!

Reply:
Done